libs/blocks/table/table.js

```diff
@@ -427,7 +427,8 @@ function applyStylesBasedOnScreenSize(table, originTable) {
       icon.parentElement.addEventListener('click', () => handleExpand(icon));
       icon.parentElement.setAttribute('tabindex', 0);
       icon.parentElement.addEventListener('keydown', (e) => {
-        e.preventDefault();
+        if (e.key !== 'Tab') e.preventDefault();
```

Comment:
This looks fine, but I'm not sure I understand the initial intent. Why do we need to prevent default? Is is just because of the fact that a Space press could scroll the page down? In that case, wouldn't it be better to prevent the default only for that key?